src/cli/commands/swarm.ts

```
@@ -487,7 +487,8 @@
 
       // Create a wrapper script that will tee the output
       const wrapperScript = `#!/bin/bash
-claude ${claudeArgs.map((arg) => `"${arg}"`).join(' ')} | tee "${agentDir}/output.txt"
+ENGINE=\"${(Deno.env.get('AUGGIE_FLOW_ENGINE') ?? 'auggie') === 'auggie' ? 'auggie' : 'claude'}\"
```

ESLint

[error] Unnecessary escape character: \".

(no-useless-escape)